packages/rum-recorder/src/transport/send.ts

```diff
@@ -15,6 +15,7 @@ export function send(endpointUrl: string, data: Uint8Array, meta: SegmentMeta):
   )
 
   toFormEntries(meta, (key, value) => formData.set(key, value))
+  formData.set('raw_segment_size', segmentSize.toString(10))
```

Comment:
This is the default, let save 2 bytes :)
```suggestion
  formData.set('raw_segment_size', segmentSize.toString())
```